backend/mora/settings.py

```diff
@@ -19,7 +19,8 @@
 LORA_URL = 'http://mox.lxc/'
 CA_BUNDLE = None
 
-AUTH = 'token'  # 'sso' or 'token'
+# Turn on SSO auth module
```

Comment:
Erm, the comment here isn't especially helpful, and seems to disagree with the code?